scripts/globals/nyzul.lua

```diff
@@ -654,7 +654,7 @@ xi.nyzul.tempBoxPickItems = function(npc)
 end
 
 xi.nyzul.tempBoxFinish = function(player, csid, option, npc)
-    local ID = require("scripts/zones/"..player:getZoneName().."/IDs")
+    local ID = require("scripts/zones/"..player:getZoneName().."/IDs") -- ?
```

Comment:
This could be replaced with `local ID = zones[player:getZoneID()]`

Reply:
Players WILL be in Nyzul though.
This function is only used once, in \scripts\zones\Nyzul_Isle\npcs\Armoury_Crate.lua

We already have ID defined at the top of the file
`local ID = require("scripts/zones/Nyzul_Isle/IDs")`

So I'm thinking this `ID` re-definition is unneccesary